dll/win32/browseui/shellbrowser.cpp

```diff
@@ -3677,12 +3688,7 @@ LRESULT CShellBrowser::OnOrganizeFavorites(WORD wNotifyCode, WORD wID, HWND hWnd
 LRESULT CShellBrowser::OnToggleStatusBarVisible(WORD wNotifyCode, WORD wID, HWND hWndCtl, BOOL &bHandled)
 {
     gSettings.fStatusBarVisible = !gSettings.fStatusBarVisible;
-    if (fStatusBar)
-    {
-        ::ShowWindow(fStatusBar, gSettings.fStatusBarVisible ? SW_SHOW : SW_HIDE);
-        RepositionBars();
-    }
-
+    CShellBrowser::RefreshGlobalUISettings();
```

Comment:
The "CShellBrowser::" prefix may not be needed (since you call that function from within a member function of that class)